spec/integration/support/inspect_and_build_examples.rb

```diff
@@ -21,12 +21,15 @@
       before(:all) do
         @subject_system = start_system(box: base)
         prepare_machinery_for_host(@machinery, @subject_system.ip, password: "vagrant")
+
+        # Enabled experimental features so that the --exclude option can be used
+        @machinery.run_command("machinery config experimental-features on", as: "vagrant")
       end
 
       it "inspects" do
         measure("Inspect") do
           @machinery.run_command(
-              "machinery inspect #{@subject_system.ip} -x --name=build_test",
+              "machinery --exclude=/packages/name=test-quote-char inspect #{@subject_system.ip} -x --name=build_test",
```

Comment:
Line is too long. [118/100]<br>Indent the first parameter one step more than the previous line.